posthog/models/team/team.py

```diff
@@ -111,6 +111,7 @@ class Team(UUIDClassicModel):
     timezone: models.CharField = models.CharField(max_length=240, choices=TIMEZONES, default="UTC")
     data_attributes: models.JSONField = models.JSONField(default=get_default_data_attributes)
     person_display_name_properties: ArrayField = ArrayField(models.CharField(max_length=400), null=True, blank=True)
+    live_events_columns: ArrayField = ArrayField(models.CharField(max_length=200), null=True, blank=True)
```

Comment:
Let's make it a `TextField`. Somebody might want to put three 60-character-long columns here, and boom, we're out of space.